diff --git a/_scripts/build.js b/_scripts/build.js
index b6bd4a4..bee3366 100644
--- a/_scripts/build.js
+++ b/_scripts/build.js
@@ -36,7 +36,7 @@ if (platform === 'darwin') {
     arch = Arch.armv7l
   }
 
-  targets = Platform.LINUX.createTarget(['deb', 'zip', '7z', 'apk', 'rpm', 'AppImage', 'pacman'], arch)
+  targets = Platform.LINUX.createTarget(['dir'], arch)
 }
 
 builder
diff --git a/_scripts/ebuilder.config.js b/_scripts/ebuilder.config.js
index c3b7e73..306867f 100644
--- a/_scripts/ebuilder.config.js
+++ b/_scripts/ebuilder.config.js
@@ -47,7 +47,7 @@ const config = {
   linux: {
     category: 'Network',
     icon: '_icons/icon.svg',
-    target: ['deb', 'zip', '7z', 'apk', 'rpm', 'AppImage', 'pacman'],
+    target: ['dir'],
   },
   // See the following issues for more information
   // https://github.com/jordansissel/fpm/issues/1503
